source/sourcing_and_processing_data/sourcing_data.rst

```diff
@@ -0,0 +1,126 @@
+* by pasting or directly writing data and uploading it into the platform
+
+.. admonition:: Supported file formats
+   :class: important
+
+   Before sourcing data and particularly files into the platform, you might want to check the formats supported by the OpenDataSoft platform (see :doc:`Supported formats</sourcing_and_processing_data/supported_formats>`).
+
+
+Uploading a file
+----------------
+
+This method consists in directly uploading a file into the platform, in order to attach a static file to a new dataset.
+
+1. In Catalog > Datasets, click on the **New dataset** button.
+2. Click on the **Add a source** button.
+3. Click on the **Upload a file** button.
+4. Choose the file to upload into the platform.
+5. Click on the **Open** button of the file selection window.
+
+.. admonition:: File size limit
+   :class: caution
+
+   The size limit for a file is 240Mo. If your file is too big you can compress it before uploading it in the platform (see :ref:`Supported compressed file formats<supportedcompressedformats>`).
+
+.. note::
+   It is possible to simply drag and drop the file after steps 1 and 2, instead of following the whole file selection procedure.
+
+
+.. _sourceremotedata:
+
+Sourcing remote data via a URL
+------------------------------
+
+This method consists in connecting the platform to:
+
+* either a remote server via its URL, to import files stored on this remote server
+* or, a remote data source exposing data records over an HTTP API
+
+.. admonition:: Scheduling updates and pushing real-time data
+   :class: note
+
+   These connections with the platform allow the automation of data update, by scheduling or pushing real-time data (see :doc:`Keeping data up to date</sourcing_and_processing_data/realtime>`).
+
+Connecting to a remote server
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. In Catalog > Datasets, click on the **New dataset** button.
+2. Click on the **Add a source** button.
+3. In the **Enter a URL** box, write the URL of the remote server where the files are stored.
+
+OpenDataSoft supports the following protocols:
+
+* http and https, which link to a single file (e.g. http://example.org/mydata.csv)
+* ftp and  ftps, which link to a single file (e.g. ftp://example.org/my_dir/mydata.csv) or to a directory (e.g. ftp://example.org/my_dir/). Using a directory is often the prefered solution to automate incremental updates between a customer's information system and the platform. All the files in the directory need to have the same format and schema (e.g. CSV files with the same column titles). In case of automation, whenever the dataset is published, new and updated files are fetched from the remote location and processed and thanks to OpenDataSoft's native deduplication strategy, similar records are not processed twice (see :doc:`Special fields documentation</sourcing_and_processing_data/dataset_schema>`).
```

Comment:
"similar records are not processed twice" they are :) But they are deduplicated, the precision looks redundant to me.